velox/core/QueryConfig.h

```diff
@@ -359,6 +359,18 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  static constexpr const char* kQueryTraceEnabled = "query_trace_enabled";
```

Comment:
How about we just use trace_ prefix? Thanks!

Reply:
Is it too broad to use the ‘trace’ prefix?